models/actions/run_job_status_test.go

```diff
@@ -59,13 +59,12 @@ func TestAggregateJobStatus(t *testing.T) {
 		{[]Status{StatusCancelled, StatusBlocked}, StatusCancelled},
 
 		// failure with other status, fail fast
```

Comment:
```suggestion
		// failure with other status, usually fail fast, but "running" wins to match GitHub's behavior
		// and we can't make "failure" wins over "running", otherwise it would cause a weird behavior that user cannot cancel a workflow or get current running workflows correctly by filter after a job fail.
```